changelogs/fragments/85623-fix-pip-check-mode-vcs.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - pip - Fix output in check_mode to properly handle VCS packages (https://github.com/ansible/ansible/issues/85592).
```

Comment:
@pkingstonxyz the diff shows ⊖ at the end. This means that there's no line feed (`\n` / LF) at the end of file. In general, it's customary to have it as it plays nicely with POSIX / *NIX envs.

Check your editor settings to make sure it doesn't strip LF at EOF.